aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -233,7 +236,7 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
         );
     }
 
-    function unlock(uint128 nonce, string memory proof) public whenNotPaused {
+    function unlock(uint128 nonce, string calldata proof) external whenNotPaused {
```

Comment:
I think it is still better to have two separate functions. This allows to expand the amount of tests that could be written for that.

Reply:
Maybe at least it will be possible to combine the `withdrawFromNear` and `withdraw` because it is really not convenient to do 3 calls to do the refund.

Reply:
Not manage to do that due to gas limit

Reply:
Ok, let's handle it in a separate pr